miniwin/miniwin/include/miniwin_d3drm.h

```diff
@@ -152,7 +152,8 @@ typedef IDirect3DRMTexture* LPDIRECT3DRMTEXTURE;
 struct IDirect3DRMTexture2 : public IDirect3DRMTexture {};
 typedef IDirect3DRMTexture2* LPDIRECT3DRMTEXTURE2;
 
-typedef struct IDirect3DRMMaterial *LPDIRECT3DRMMATERIAL, **LPLPDIRECT3DRMMATERIAL;
+struct IDirect3DRMMaterial {};
```

Comment:
I'm surprised this does not need to inheris `IUnknown`. How is it freed?

Reply:
Oops. Its a good question though, my initial thinking was that there is a lot of leakage, but I guess it's just passed to another com object which will cause it to get freed when that one is freed, we don't do so yet

Reply:
When in doubt, can we add this? Along the line, I think we want to rely on correct clean-up by the reference counts.

Reply:
It's on it's way :)